tools/add_to.hpp

```diff
@@ -140,52 +140,51 @@ namespace antler {
             ->transform(escape_transform);
 
          lib_subcommand = subcommand->add_subcommand("lib", "Add a new library to your project.");
          lib_subcommand->footer(std::string(R"(Examples:)")
                + "\n\t" + app.get_name() +R"( add lib MyLib C++ \\-O2 "\-s")"
                + "\n\t" + app.get_name() +R"( ./path-to-project/ add lib -n MyLib -l C++ --comp -O2 --link -s)");
-         lib_subcommand->add_option("-p", path, "Path containing the project's yaml file.");
-         lib_subcommand->add_option("-n, name", obj_name, "The name of the library to add.")->required();
-         lib_subcommand->add_option("-l, lang", lang, "Language this library will use.")->required();
-         lib_subcommand->add_option("--comp, compile_options", copts, "Options for the compiler for this app.")
+         lib_subcommand->add_option("-p, --path", path, "Path containing the project's yaml file.")->default_val(".");
+         lib_subcommand->add_option("-n, --name", obj_name, "The name of the library to add.")->required();
+         lib_subcommand->add_option("-l, --lang", lang, "Language this library will use.")->required();
+         lib_subcommand->add_option("--comp", copts, "Options for the compiler for this app.")
             ->transform(escape_transform);
-         lib_subcommand->add_option("--link, link_options", lopts, "Options for the linker for this app.")
+         lib_subcommand->add_option("--link", lopts, "Options for the linker for this app.")
             ->transform(escape_transform);
 
          dep_subcommand = subcommand->add_subcommand("dep", "Add a new dependency to the project.");
          dep_subcommand->footer(std::string(R"(Examples:)")
                + "\n\t" + app.get_name() +R"( add dep MyApp MyDep)");
-         dep_subcommand->add_option("-p", path, "Path containing the project's yaml file.");
-         dep_subcommand->add_option("-o, obj", obj_name, "The name of the object to attach dependency to.")->required();
-         dep_subcommand->add_option("-d, dep", dep_name, "The name of the dependency.")->required();
-         dep_subcommand->add_option("-l, loc", location, "Location of the dependency.");
-         dep_subcommand->add_option("-t, tag", tag, "Tag associated with the dependency.");
-         dep_subcommand->add_option("-r, release", release, "Release version of the depedency.");
-         dep_subcommand->add_option("--digest, hash", hash, "Hash of the dependency.");
+         dep_subcommand->add_option("-p, --path", path, "Path containing the project's yaml file.")->default_val(".");
+         dep_subcommand->add_option("-o, --obj_name", obj_name, "The name of the object to attach dependency to.")->required();
+         dep_subcommand->add_option("-d, --dep_name", dep_name, "The name of the dependency.")->required();
+         dep_subcommand->add_option("-l, --lang", location, "Location of the dependency.");

```

Comment:
Sorry, missed this in my previous review.
```suggestion
         dep_subcommand->add_option("-l, --loc", location, "Location of the dependency.");
```
or
```suggestion
         dep_subcommand->add_option("-l, --location", location, "Location of the dependency.");
```

Reply:
I find confusing that `-l` is language for `app` and `lib` but location for `dep`, thinking on solution.. maybe `--url` instead of location?

Reply:
also, I assume we can deduct dependency name from location and make it optional